src/components/Icons/SpeechBubbleIcon.tsx

```diff
@@ -4,9 +4,10 @@ import { faComment } from "@fortawesome/free-solid-svg-icons";
 
 interface Props {
     onHoverText?: string;
+    color?: string;
 }
 const SpeechBubbleIcon: React.FC<Props> = (props) => {
-    return <Icon icon={faComment} onHoverText={props.onHoverText} color="blue" />;
+    return <Icon icon={faComment} onHoverText={props.onHoverText} color={props.color} />;
```

Comment:
why this change? I can't see where you've used this, but I don't see why it 'shouldn't be blue - can understand making this the _theme_ blue (rather than 'hardcoded' blue) but not having it just be 'any colour specified'